app/helpers/datasets_helper.rb

```diff
@@ -62,15 +62,43 @@ def show_more?(datafiles, datafile)
     "js-show-more-datafiles" unless datafiles.take(5).include? datafile
   end
 
-  def contact_details_exist?(dataset)
-    contact_email_exists?(dataset) || foi_email_exists?(dataset)
+  def contact_information_exists?(dataset)
+    contact_email_exists?(dataset) || foi_details_exist?(dataset)
   end
 
   def contact_email_exists?(dataset)
     dataset.contact_email.present? || dataset.organisation.contact_email.present?
```

Comment:
We're duplicating the logic in `contact_email_for`. We can change this to:

```rb
def contact_email_exists?(dataset)
  contact_email_for(dataset).present?
end
```

if we add the `.presence` call to both conditions in `contact_email_for`:

```rb
def contact_email_for(dataset)
  dataset.contact_email.presence || dataset.organisation.contact_email.presence
end
```